### PR TITLE
IAP-3685: update API Portal docker documentation ...

### PR DESCRIPTION
**Pull request by kvedenskii**
* update apiportal docker intro section
* update run API Portal container from ready-made image
* add docker configuration options to run container section
* modify troubleshooting section
* refactor links

https://deploy-preview-1429--axway-open-docs.netlify.app/docs/apim_installation/apiportal_docker/

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Passes all status checks
* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_installation/apiportal_docker/_index.md [hunk 1]
```diff
@@ -7,16 +7,16 @@
     "description":"Deploy API Portal in Docker containers and learn how to upgrade a container deployment."
 }
 
-[Docker](https://www.docker.com/) containers are software containers that wrap software in a complete file system containing everything the software in question needs to run properly, such as code, runtime, system tools, and system libraries. This guarantees that the software always runs the same way, regardless of environment. In addition, you can configure several containers to run on a single host machine, without the overhead of starting and maintaining separate virtual machines.
+[Docker][docker home] containers are software containers that wrap software in a complete file system containing everything the software in question needs to run properly, such as code, runtime, system tools, and system libraries. This guarantees that the software always runs the same way, regardless of environment. In addition, you can configure several containers to run on a single host machine, without the overhead of starting and maintaining separate virtual machines.
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/_index.md:10]: @kvedenskii What's it?

File: content/en/docs/apim_installation/apiportal_docker/_index.md [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
 
-Docker containers enable you to run API Portal on any host operating system or cloud platform that supports Docker. For details on how to deploy your Docker infrastructure in the cloud, see the product documentation for your chosen third-party cloud platform or Platform as a Service (PaaS) environments, such as [Amazon Web Services](https://aws.amazon.com/).
+Docker containers enable you to run API Portal on any host operating system or cloud platform that supports Docker. For details on how to deploy your Docker infrastructure in the cloud, see the product documentation for your chosen third-party cloud platform or Platform as a Service (PaaS) environments, such as [Amazon Web Services][aws home].
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/_index.md:12]: @kvedenskii "[Amazon Web Services][aws home]." -> ??

File: content/en/docs/apim_installation/apiportal_docker/_index.md [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
 
 Because API Gateway and API Manager also support Docker containers, you can run the full API Management solution in Docker containers. This makes it easy to quickly build, test, and run API Management solutions for standalone or high availability (HA) deployment.
 
 For more details on Docker, see the following:
 
-* [Docker overview](https://docs.docker.com/engine/understanding-docker/)
-* [Install Docker on Linux distributions](https://docs.docker.com/engine/installation/linux/)
+* [Docker overview][docker overview]
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/_index.md:18]: @kvedenskii why did you remove the links?

File: content/en/docs/apim_installation/apiportal_docker/_index.md [hunk 4]
```diff
@@ -7,16 +7,16 @@
...
+* [Install Docker][install docker]
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/_index.md:19]: @kvedenskii why did you remove the links?

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md [hunk 5]
```diff
@@ -8,18 +8,15 @@
...
 The following components are required on your system before you can deploy API Portal in Docker containers:
 
-* Docker version 1.13.x
-* MySQL Docker container running
-* API Portal Docker image available from Axway Support at [https://support.axway.com](https://support.axway.com/)
-* API Gateway and API Manager either installed on-premise or deployed in containers (see the [API Gateway Installation Guide](/docs/apim_installation/apigtw_install/) for more details)
+* [Docker engine][docker engine]
+* MySQL server
+* API Portal Docker image available from [Axway Support][axway support]
+* API Gateway and API Manager either installed on-premise or deployed in containers (see the [API Gateway Installation Guide](/docs/apim_installation/apigtw_install/) for more details). API Gateway and API Manager are not required to be installed in order to run API Portal, you can install them and configure in API Portal later
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md:18]: @kvedenskii Can you please clarify this:

- The following components are required: API Gateway and API Manager...
- API Gateway and API Manager are not required to be installed..

**Review by andreamussap — APPROVED** [review]

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md (hunk 5 above)
Comment (andreamussap) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md]: @kvedenskii @dginevaAxway @jowens-axway  This PR is merged, but please review and clarify this in a new PR. Thanks
